ISSUE_TEMPLATE.md

```diff
@@ -1,88 +1,9 @@
-    - [ ] Write (code + tests + docs) then get them merged.  **ALL-PR-NUMBERS**
-      - [ ] **Code needs to be disabled by default.**   Verified by code OWNERS
-      - [ ] Minimal testing
-      - [ ] Minimal docs
-        - cc `@kubernetes/docs` on docs PR
-        - **cc `@kubernetes/feature-reviewers` on this issue** to get approval before checking this off
-        - New apis: *Glossary Section Item* in the docs repo: kubernetes/kubernetes.github.io
-      - [ ] Update release notes
-- [ ] Beta
-  - [ ] Testing is sufficient for beta
-  - [ ] User docs with tutorials
-        - *Updated walkthrough / tutorial* in the docs repo: kubernetes/kubernetes.github.io
-        - cc `@kubernetes/docs` on docs PR
-        - **cc `@kubernetes/feature-reviewers` on this issue** to get approval before checking this off
-  - [ ] Thorough API review
-    - **cc `@kubernetes/api`**
-- [ ] Stable
-  - [ ] docs/proposals/foo.md moved to docs/design/foo.md 
-        - **cc `@kubernetes/feature-reviewers` on this issue** to get approval before checking this off
-  - [ ] Soak, load testing 			
-  - [ ] detailed user docs and examples
-    - **cc `@kubernetes/docs`**
-    - **cc `@kubernetes/feature-reviewers` on this issue** to get approval before checking this off
-
-*FEATURE_STATUS is used for feature tracking and to be updated by `@kubernetes/feature-reviewers`.*
-**FEATURE_STATUS: IN_DEVELOPMENT**
-
-More advice:
-
-Design
-   - Once you get LGTM from a *`@kubernetes/feature-reviewers`* member, you can check this checkbox, and the reviewer will apply the "design-complete" label.
- 
-Coding
-  - Use as many PRs as you need.  Write tests in the same or different PRs, as is convenient for you.
-  - As each PR is merged, add a comment to this issue referencing the PRs.  Code goes in the http://github.com/kubernetes/kubernetes repository,
-        and sometimes http://github.com/kubernetes/contrib, or other repos.
-  - When you are done with the code, apply the "code-complete" label.
-  - When the feature has user docs, please add a comment mentioning `@kubernetes/feature-reviewers` and they will
-        check that the code matches the proposed feature and design, and that everything is done, and that there is adequate
-        testing.  They won't do detailed code review: that already happened when your PRs were reviewed.
-        When that is done, you can check this box and the reviewer will apply the "code-complete" label.
-
-Docs
-  - [ ] Write user docs and get them merged in.
-  - User docs go into http://github.com/kubernetes/kubernetes.github.io.
-  - When the feature has user docs, please add a comment mentioning `@kubernetes/docs`.
-  - When you get LGTM, you can check this checkbox, and the reviewer will apply the "docs-complete" label.
-
+# Feature Description
+- One-line feature description (will be used as a release note):
```

Comment:
Some features require longer release notes.

I don't want to gate this PR on it, but we need to change our release-note processes, agree on a user-friendly release-note structure, make it clearer what release notes should contain (e.g., actionable information, links to docs about new features, names of fields or flags added/removed/changed, remediating actions), examples of good release notes, removal of the release-note template in the kubernetes repo, bot support to prompt for appropriate release notes and/or removal of the release-note gate in the kubernetes repo.

We also need to distinguish features targeted for specific releases and multi-release roadmap placeholders for larger areas of work, which we do need to track somehow. 

Examples of the former might be:
* PDB beta and GA: https://github.com/kubernetes/kubernetes/issues/25321
* Taints/tolerations GA: https://github.com/kubernetes/kubernetes/issues/25320

Examples of the latter are:
* Workloads APIs GA: https://github.com/kubernetes/kubernetes/issues/42752
* Moving kubectl logic to the system: https://github.com/kubernetes/kubernetes/issues/12143